### ISSUE TITLE
In and Up asymptotic amplitudes

### ISSUE THREAD

**Issue by DSSystems** [BlackHolePerturbationToolkit/ReggeWheeler#7]
Perhaps it is a bit late but I finished adapting the code I use to calculate the incidence and reflection coefficients for the In and Up solutions. I just modified the current MST.m file and added (I hope without breaking anything else) my code.

**Comment by barrywardell** [comment]
Thank you. We still don't have the asymptotic amplitudes so this would be great to have. Where can we find the updated MST.m?

**Comment by DSSystems** [comment]
Dear Barry,

I am attaching the (compressed & modified) MST.m file. Let me know if you find an error or the code is unclear.

Cheers, David.

[MST.m.zip](https://github.com/BlackHolePerturbationToolkit/ReggeWheeler/files/4985035/MST.m.zip)

**Comment by barrywardell** [comment]
Thanks. I've been checking these today by comparing against numerical evaluation of the radial functions in the asymptotic regimes. Everything seems to be correct with the exception of the complex argument of Cref. I find that it is off by an s-dependent amount (for all cases omega=0.6, l=2):

s=0: -2.5318
s=1: -2.0365
s=2: 2.3487

Do you have any idea what could be wrong?

**Comment by barrywardell** [comment]
I think I have found the problem. Whereas the code had:
```Mathematica
UpRef = -(Conjugate[UpTra]/Conjugate[InTra])InRef
```
it should have been
```Mathematica
UpRef = -(Conjugate[InRef]/Conjugate[InTra])UpTra
```
Changing this makes everything agree with the numerics. Does is seem right to you?

**Comment by DSSystems** [comment]
I just checked my notes and you are right. I accidentally swapped the conjugation when I wrote the code!

**Comment by barrywardell** [comment]
Great, thanks for confirming. This has now been merged into master and included in the 0.3.0 release version.
